src/tag/constants.js

```diff
@@ -6,13 +6,13 @@ LICENSE file in the root directory of this source tree.
 */
 // @flow
 /* eslint-disable import/prefer-default-export */
-import DEFAULT_THEME from '../themes/light-theme';
+import {LightTheme} from '../themes';
 
 export const STYLE = {
-  primary: DEFAULT_THEME.colors.primary400,
-  warning: DEFAULT_THEME.colors.warning400,
-  positive: DEFAULT_THEME.colors.positive400,
-  negative: DEFAULT_THEME.colors.negative400,
+  primary: LightTheme.colors.primary400,
+  warning: LightTheme.colors.warning400,
+  positive: LightTheme.colors.positive400,
+  negative: LightTheme.colors.negative400,
```

Comment:
Not related to this PR, but it needs to be changed to a string like `primary400 `, `warning400 `, etc, and then get the values from a `$theme` in a styled component. This way the LightTheme wouldn't be the default but whatever theme is passed to the Provider.